OpenBCI/OpenBCI_GUI
# Fix Widget Selector Drawing Error

- **Comment out `delay(1000)` in WidgetManager.pde**


- Appropriately size the widget selector at the right times to remove extra grey space being drawn.

- Minor refactor of Widget.pde to include appropriate access modifiers

Before, only looked right when widget dimensions were recalculated. Now, looks right the first time, and every time.

Test WidgetSelector:
- Clear all settings (only happens with default settings)
- Open GUI synthetic mode or Cyton mode
- Don't resize the GUI or move any widgets
- Hover over the Widget Selector dropdown in the TimeSeries (full height container)
- Check that there is no grey space

Test Widget Refactor:
- Aggressively test the WidgetSelector dropdown for all widgets in multiple containers


_____


**Before**
![Screen Shot 2020-05-18 at 9 25 37 PM](https://user-images.githubusercontent.com/25914123/82277735-3ee64a80-994e-11ea-87fc-b208a8128b4e.png)

**After**
![Screen Shot 2020-05-18 at 10 21 00 PM](https://user-images.githubusercontent.com/25914123/82281046-f6cb2600-9955-11ea-8390-9fca9ad8db19.png)
